std/experimental/allocator/mallocator.d

```diff
@@ -64,19 +64,19 @@ struct Mallocator
 }
 
 ///
-unittest
+@nogc unittest
 {
     auto buffer = Mallocator.instance.allocate(1024 * 1024 * 4);
     scope(exit) Mallocator.instance.deallocate(buffer);
     //...
 }
 
-unittest
+@nogc unittest
 {
-    static void test(A)()
+    @nogc static void test(A)()
     {
         int* p = null;
-        p = new int;
+        p = cast(int*) A.instance.allocate(int.sizeof);
```

Comment:
These `test` functions appear to have been left in a state of flux. The one in the unittest below this one still appears completely nonsensical.